homeassistant/components/fitbit/manifest.json

```diff
@@ -3,7 +3,9 @@
   "name": "Fitbit",
   "documentation": "https://www.home-assistant.io/components/fitbit",
   "requirements": [
-    "fitbit==0.3.0"
+    "fitbit==0.3.0",
```

Comment:
fitbit 0.3.1 requires requests-oauthlib>=0.7 and at the moment should this lead to the installation of requests-oauthlib 1.2.0. requests-oauthlib requires oauthlib>=3.0.0.

I would suggest to upgrade fitbit to 0.3.1 and let it handle its dependencies.

Reply:
Sounds reasonable.